app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb

```diff
@@ -32,9 +32,13 @@ insmod fat
 insmod chain
 
 echo "VMWare hosts with QuickBoot feature enabled may not find the local ESP"
-echo "partition due to not initializing all the EFI devices. To workaround, upgrade"
-echo "to the latest grub2 (*) and uncomment "connectefi scsi" statement in the"
-echo "grub2_chainload template."
+echo "partition due to not initializing all the EFI devices. To address this,"
+echo "use an up-to-date grub2 (*) version and include the "connectefi scsi" statement"
+echo "as provided below. If you're using an older grub2 version or"
+echo "the "connectefi" option isn't recognized by your grub2, grub2 will print a error"
+echo "like 'can't find command connectefi' but the boot process will continue."
+echo "You can omit this by adding a (global) parameter to 'grub2-connectefi=false'."
```

Comment:
Actually setting a global parameter would not omit the error in the default template.

Reply:
Do you have a suggestion how we should describe it then?

Reply:
I think something along the lines of:
```
To omit it from the default GRUB2 script, remove the "@host.nil?" statement from the template.
To omit it for a specific host configuration, set grub2-connectefi parameter to false.
```
How does that sound to you?